packages/core-parse/src/extensions/spans.js

```diff
@@ -0,0 +1,50 @@
+/** Roles for wrapping inline text in containers with semantic meaning. */
+import { u } from 'unist-builder'
+
+import { RoleProcessor } from '../roleProcessor.js'
+
+export const spansExtension = {
+    name: 'spans',
```

Comment:
`inlineMarkup` extensions? Calling this span is strange to me, that means `<span>`, which isn't what these are? flowContent?

Reply:
Yeh I literally just made this name up before our call, and happy to change. `flowContent` doesn't seem very "human intuitive" though 🤔

Reply:
What about `inline` and `inlineExtensions` -- seems a bit general, but better than span?

Reply:
yeh, renamed to `inlineMarkup` 👍